beacon-chain/blockchain/execution_engine.go

```diff
@@ -316,7 +316,7 @@ func (s *Service) getPayloadAttribute(ctx context.Context, st state.BeaconState,
 
 	var attr payloadattribute.Attributer
 	switch st.Version() {
-	case version.Capella:
+	case version.Capella, version.Deneb:
```

Comment:
we'll also want to add this to events.go streamPayloadAttributes line 320 to handle deneb

Reply:
[3021b2d](https://github.com/prysmaticlabs/prysm/pull/12509/commits/3021b2d4d8ddb711782d45b3bfa23494098988ab)